agent/discoveryTools.mjs

```diff
@@ -61,7 +61,8 @@ export function initialSkaleNetworkScanForS2S() {
                 "secondsToWaitForSkaleNetworkDiscovered":
                     imaState.optsS2S.secondsToWaitForSkaleNetworkDiscovered,
                 "chain": imaState.chainProperties.sc,
-                "bParallelModeRefreshSNB": imaState.optsS2S.bParallelModeRefreshSNB ? true : false
+                "bParallelModeRefreshSNB": imaState.optsS2S.bParallelModeRefreshSNB ? true : false,
```

Comment:
could be replaced with `!!imaState.optsS2S.bParallelModeRefreshSNB`

Reply:
yes, agree

Reply:
Fixed in many places, thanx